vassal-app/src/main/java/VASSAL/build/module/NotesWindow.java

```diff
@@ -273,23 +275,26 @@ public static String getConfigureTypeName() {
   public void addTo(Buildable b) {
     super.addTo(b);
     getLaunchButton().setAlignmentY(0.0F);
-    GameModule.getGameModule().addCommandEncoder(this);
-    GameModule.getGameModule().getGameState().addGameComponent(this);
-    GameModule.getGameModule().addCommandEncoder(privateNotes);
-    GameModule.getGameModule().getGameState().addGameComponent(privateNotes);
-    GameModule.getGameModule().addCommandEncoder(secretNotes);
-    GameModule.getGameModule().getGameState().addGameComponent(secretNotes);
+    final GameModule gm = GameModule.getGameModule();
+    gm.addCommandEncoder(this);
+    gm.getGameState().addGameComponent(this);
+    gm.addCommandEncoder(privateNotes);
+    gm.getGameState().addGameComponent(privateNotes);
+    gm.addCommandEncoder(secretNotes);
+    gm.getGameState().addGameComponent(secretNotes);
+    gm.addKeyStrokeSource(new KeyStrokeSource(frame.getRootPane(), JComponent.WHEN_IN_FOCUSED_WINDOW));
```

Comment:
Is this introducing a leak, as you don't call `removeKeyStrokeSource()` in `removeFrom()` below?

Reply:
There is no `removeKeyStrokeSource()` to call in GameModule. I think you're thinking of the one you made for the listeners so that they would de-register?

Reply:
I thought I had made one of those. So, is this going to be a problem?